templates.tf

```diff
@@ -56,16 +56,7 @@ data "template_file" "hms_readonly" {
     hive_metastore_log_level = "${var.hms_log_level}"
     nofile_ulimit            = "${var.hms_nofile_ulimit}"
     enable_metrics           = "${var.enable_hive_metastore_metrics}"
+    shared_schemas            = "${join(",",var.apiary_shared_schemas)}"
     instance_name            = "${local.instance_alias}"
-
-    ranger_service_name       = "${local.instance_alias}-metastore"
-    ranger_policy_manager_url = "${var.ranger_policy_manager_url}"
```

Comment:
To match schema access configuration same as S3 buckets.
Since access to read-only metastore access is primarily controlled at account level, in practice this makes it simpler, so that we don't have to know which AD groups are users coming in through read-only metastore part of.

Reply:
okay.